### ISSUE TITLE
popgenWindows only runs on some scaffolds

### ISSUE THREAD

**Issue by emgiles** [simonhmartin/genomics_general#110]
Hi Simon

Sorry to bother! I am running popgenWindows on my geno file created from an all sites vcf with around 30 million variants. My reference genome has about 1000 scaffolds. When I run popgenWindows, I only get estimates for a small number of scaffolds. I have tried running popgenWindows with -m 10000 (results for less than 6 scaffolds), -m 1000 (results for 145 scaffolds), and -m 100 (results for 162 scaffolds). Furthermore, the results obtained with the different values of m don't depend on scaffold size, rather scaffold id, ie results are given for scaffolds 1 -145 even though scaffold 864 is very large. I suppose I can just split my vcfs by scaffold and start over, but I am wondering why the program is only giving results for some scaffolds and if there is a way around this. Thank you!

**Comment by simonhmartin** [comment]
Hi, I'm not sure why this is happening. I suggest including the option `--writeFailedWindows`. This will create an output for every window attempted, and you will be able to see how many sites were available for analysis in each window.

**Comment by emgiles** [comment]
Hi again,
I subsetted my vcf to only include the 10 largest scaffolds. When I ran popgenWindows with these, again, I only retrieved statistics for approx 3300 windows (scaffolds 1 and 2). The program just failed to run on the remaining scaffolds, no error was given. I unzipped the .geno file to check that all 10 scaffolds are present, I ran popgenWindows on the unzipped file and now it has produced stats for all 10 scaffolds (approx 17,000 windows). Not sure why, but the problem was fixed by unzipping geno.
